services/director/src/simcore_service_director/producer.py

```diff
@@ -549,7 +548,10 @@ def _to_datetime(datetime_str: str) -> datetime:
     last_task_error_msg = (
         last_task["Status"]["Err"] if "Err" in last_task["Status"] else ""
     )
-    if task_state in ("failed"):
+    if "Err" in last_task["Status"]:
+        log.error("service %s failed with %s", service_name, last_task["Status"])
+        last_task_state = ServiceState.FAILED
+    elif task_state in ("failed"):
```

Comment:
just wondering: how do you account for the fact that a service may be able to start later when some resources come back?

Reply:
Not doing this. It will fail as soon as possible.
The previous implementation let everything pending forever, without informing the user.

Is making the services fail for lack of resources not desirable?